Dockerfile

```diff
@@ -11,7 +11,7 @@ RUN apk update && \
     yarn cache clean
 
 # Runtime container with minimal dependencies
-FROM node:16-alpine
+FROM node:18-alpine
```

Comment:
should line 29 (CMD) be `__build__/src/index.js` ?

Reply:
it actually doesnt even run that command

Reply:
interesting :o